src/bitmessagekivy/mpybit.py

```diff
@@ -1,34 +1,338 @@
+# pylint: disable=unused-import, too-many-public-methods, unused-variable, too-many-ancestors
```

Comment:
do we really need all of these?

Reply:
I will fix some but some disable will be needed.